HGGM.IntegrationTests/StartupTests.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc.Testing;
+using Xunit;
+
+namespace HGGM.IntegrationTests
+{
+    public class StartupTests : IClassFixture<WebApplicationFactory<Startup>>
+    {
+        private readonly WebApplicationFactory<Startup> _factory;
+
+        public StartupTests(WebApplicationFactory<Startup> factory)
+        {
+            _factory = factory;
+        }
```

Comment:
`IClassFixture` injects a `WebApplicationFactory`, which takes care os starting the server so the integration test can run. It provides a `HttpClient`